### DISCUSSION TITLE
In the past, there was a command/setting that would fully eliminate the minimize animation - this seems gone in Big Sur?

### DISCUSSION THREAD

**Discussion by DolfLund** [yannbertrand/macos-defaults#136] [Q&A]
For some reason BigSur without full graphics acceleration has a lot more trouble with the minimize window effect then Catalina did (for example in a VM) - delays approach 15 seconds if you accidentally minimize. I am assuming that the old command no longer works to fully disable the animation.  It also seems changing the speed of it is also not possible. Just wondering if someone who better understands what is occurring might know if this is a permanent limitation of Big Sur or something that might be overcome.

Thanks!

**Comment by yannbertrand** [comment]
Do you remember what the command was?

I don't think it can help but did you try [`defaults write com.apple.dock mineffect -string scale`](https://macos-defaults.com/dock/mineffect.html#set-to-scale)?

**Reply by DolfLund** [reply]
It helped by reducing the lag, thank you - now it takes about 10 seconds to minimize (vs 15-20) and less than 6 seconds to re-expand (vs 15-20).  It's quite helpful, but of course not a resolution as my workflow still needs to avoid using that button - but thanks for the suggestion.
